pilot/pkg/features/pilot.go

```diff
@@ -477,6 +477,12 @@ var (
 	// New behavior (true): we create listener 0.0.0.0_8080 and route http.8080. This has no conflicts; routes are 1:1 with listener.
 	UseTargetPortForGatewayRoutes = env.RegisterBoolVar("PILOT_USE_TARGET_PORT_FOR_GATEWAY_ROUTES", true,
 		"If true, routes will use the target port of the gateway service in the route name, not the service port.").Get()
+
+	VerifyCertAtClient = env.RegisterBoolVar(
```

Comment:
This needs to get passed down. It should be stored in a struct (haven't looked at that code in a while).